deployments/helm/nvidia-device-plugin/values.yaml

```diff
@@ -118,7 +118,7 @@ gfd:
 # Helm dependency
 nfd:
   nameOverride: node-feature-discovery
-  enableNodeFeatureApi: false
+  enableNodeFeatureApi: true
```

Comment:
Does the behaviour differ between upgrades and new installations due to this toggle?

Reply:
@rajathagasthya have you tried a helm chart upgrade? What behavior have you observed?